Release Notes 17.1 (#21835)
opf/openproject: docs/release-notes/17-1-0/README.md

sidebar_navigation:
    title: 17.1.0
release_version: 17.1.0
release_date: 2026-02-11
---

 # OpenProject 17.1.0

 Release date: 2026-02-11

 We released [OpenProject 17.1.0](https://community.openproject.org/versions/2237). The release contains several bug fixes and we recommend updating to the newest version. In these Release Notes, we will give an overview of important feature changes. At the end, you will find a complete list of all changes and bug fixes.

<!-- BEGIN CVE SECTION -->

<!-- END CVE SECTION -->
## Important feature changes

### Automated project initiation request with a guided wizard (Enterprise add-on)

> OpenProject introduces a configurable wizard for project initiation requests. The wizard is available as an **Enterprise add-on in the Premium plan** and can be enabled per project template.

Reviewer: ```suggestion
OpenProject introduces a configurable wizard for project initiation requests. The wizard can be enabled per template project.

[feature: project_creation_wizard ]
```

Reviewer: @corinnaguenther let's use the Enterprise Add-on banners in the release notes as well, to harmonize the documentation look.

Author: @MayaBerd Great idea, I'll just have to get used to this and remember it for the future 👍

#### Work package created for a project initiation request

When a project initiation request is submitted, OpenProject automatically creates a work package that represents the request and serves as its central tracking artifact.

The work package is created based on the wizard configuration, including work package type, status, and assignee. Assignment can be derived from a project attribute or a project role, such as a project owner. The work package contains links back to the wizard and can be processed using existing workflows.

![OpenProject work package of type "Project initiation request" with a description including a link to the artifact, assignee and accountable and a comment automatically generated by the system.](openproject-project-initiation-request-work-package.png)

#### Automatically generated project initiation artifact (PDF)

Upon submission of the project initiation request, a PDF artifact is automatically generated. The artifact contains all information entered in the wizard and is attached to the corresponding work package for documentation and audit purposes.

> The artifact is updated automatically whenever the status of the project initiation request work package changes, ensuring that the documentation always reflects the current state.

Reviewer: That was also my understanding initially, but after a call with Henriette yesterday I am not sure. @ulferts is in fact the same single PDF updated automatically, or a new one (with a time stamp for versioning) created after every change to the request?

Author: Very important note, thank you, Maya. I just tested it and the work package doesn't seem to update after the first submission – apart from adding a new PDF to the files section.
@ulferts Is the following the expected behavior?

_When a project initiation request is submitted for the first time, OpenProject automatically creates a work package that represents the request and serves as its central tracking artifact.

The work package is created once based on the wizard configuration at the time of the initial submission, including the work package type, status, and initial assignee. The assignee can be derived from a project attribute or a project role, such as a project owner.

>[!NOTE]
> If the project initiation request is submitted again later, the changes (like different assignee) might not be automatically displayed in the work package and need to be manually updated. However, each submission generates a new PDF artifact, which is uploaded and linked to the existing work package. This allows changes to be reviewed and documented over time, while keeping the work package as the central place for processing the request using existing workflows._

Author: FYI: I will merge this PR so that I can share it internally. Will still do updates on the Release Notes next week, if needed.

![OpenProject meeting showing option to duplicate an agenda item to the next meeting occurrence by clicking the More menu of an agenda item --> Duplicate --> Duplicate in next occurrence](openproject-17-1-meetings-duplicate-agenda-item.png)

### Release Attribute highlighting to Community

The Attribute highlighting feature, previously available only as an Enterprise add-on, is now included in the free Community plan.

Users can configure attribute highlighting in work package table views by opening the table configuration and selecting the **Highlighting** tab. Attributes such as Status, Priority, and Finish date can be highlighted inline or applied as full-row highlights based on attribute values. This makes key attributes visually distinguishable directly in the work package list without opening individual items.

[Read more about attribute highlighting in our documentation](../../user-guide/work-packages/work-package-table-configuration/#attribute-highlighting).

Here's an example of highlighting work packages by priority:

![A work package table in OpenProject, highlighted by Priority](openproject-attribute-highlighting-priority.png)

> ### Warning before opening external links in user-provided content (Enterprise add-on)

Reviewer: I am not sure about the heading here. 
The warning itself is not the Enterprise add-on, but part of it. 
Btw, the link is: docs/system-admin-guide/system-settings/external-links/ (thought the name in the banner and the page need to be aligned, the URL should stay the same)

Author: @MayaBerd Not sure if I understand it then. Do you mean that the Enterprise add-on is the setting, not the warning? So "Enable a warning before opening external links in user-provided content (Enterprise add-on)" would work?

Reviewer: @corinnaguenther "Enable a warning before opening external links in user-provided content (Enterprise add-on)" is definitely more descriptive and would make it easier for me to understand it. 
I was thinking more of using the same name throughout all mentions (Enterprise guide, pricing page, documentation page). The application says "External links", banner says "Capture External Links" at the moment. I would say let's see what the decision on the banner will be and then harmonize it.

> OpenProject introduces a warning before opening external links in user-provided content, available as an **Enterprise add-on in the Premium plan**. The warning adds an additional security layer by making users aware that they are about to leave OpenProject.

Reviewer: ```suggestion
[feature: capture_external_links ]

With 17.1 OpenProject introduces an option to add a warning when accessing external links from formatted text, such as project descriptions, comments or wiki pages. The warning adds an additional security layer by making users aware that they are about to leave OpenProject.
```

When users click on an external link, a confirmation dialog is displayed indicating that the link leads outside of OpenProject. This applies to links added by users, for example in descriptions, comments, or other text fields. In SaaS trial environments, external link handling is enforced to ensure that warnings for user-provided external links cannot be bypassed.

![Warning in OpenProject: "You are about to leave OpenProject and visit an external website. Please be aware that external websites are not under our control and may have different privacy and security policies. Are you sure you want to proceed to the following external link?" with button "continue to external website"](openproject-17-1-warning-external-link.png)

> ### Show short and weight values for Hierarchy and Weighted item list fields

Reviewer: These fields are also an Enterprise add-on, I think normally you add (Enterprise add-on) in the heading?

Author: Right, of course 👍

Users now see an item’s short or weight wherever values from Hierarchy or Weighted item list custom fields (work packages) and project attributes are shown. This provides an extra hint to confirm that the right item was selected.

This information is displayed in:
- Work package details (for assigned values).
- Work package tables (for assigned values).
- Project attributes (for assigned values).
- The tree view selector while selecting a value (work packages and projects).
- The admin Items tab when editing the field, including the tree overview.

![OpenProject administration for custom fields, example of "Bending factor" for different materials, each one displayed with a specific value in brackets, e.g. "Aluminium (0.4)".](openproject-17-1-custom-fields-values.jpg)

> 

Reviewer: I would also mention other UX/UI changes in one section
- Primerization of Access tokens section under Account settings
- Primerization of Backlogs section under System administration
- Primerization of password confirmation dialogue
- Showing breadcrumb with full project hierachy in Project Overview showing portfolios and programs
- Rename Nextcloud GroupFolder references to TeamFolder - though this might be a less noticeable one
